styles.css

```diff
@@ -538,4 +620,338 @@
+    word-spacing: 4px;
+    line-height: 28px;
+  }
+
+  .card-text-1,
+  .btn-hide {
+    display: none;
+  }
+
+  .css-list-big {
+    display: inline-block;
+  }
+
+  .btns-3,
+  .btns-33 {
+    margin-left: 20px;
+    margin-top: 20px;
+    margin-bottom: 40px;
+  }
+
+  .btns-33 {
+    margin-left: 0;
+  }
+
+  .btn-11 {
+    margin-left: 0;
+  }
+
+  .img-hide,
+  .title-hide,
+  .canopy-show,
+  .text-hide {
+    display: none;
+  }
+
+  .img-big {
+    display: block;
+    width: 600px;
+    height: 500px;
+    margin-right: 40px;
+    margin-top: 7px;
+    margin-left: 7px;
+  }
+
+  .title-show,
+  .canopy-hide,
+  .text-show,
+  .btn-show {
+    display: block;
+  }
+
+  .btn1-invert {
+    background-color: #4053fc;
+    color: #fff;
+    margin-left: 0;
+  }
+
+  .btn1-invert:hover {
+    background-color: #fff;
+    color: #4053fc;
+  }
+
+  .btn1-invert:visited {
+    background-color: #e5e5e5;
+    color: #5e6c84;
+  }
+
+  .hand {
+    display: block;
+    position: relative;
+    bottom: 40px;
+    right: -430px;
+  }
+
+  .third-hide,
+  .about-hide {
+    display: none;
+  }
+
+  .third-show {
+    display: block;
+  }
+
+  .about-show {
+    display: flex;
+    margin-top: 40px;
+    padding: 130px;
+    background-color: #fff;
+    border-top-right-radius: 80px;
+  }
+
+  .left {
+    flex-basis: 50%;
+  }
+
+  .right {
+    flex-basis: 50%;
+  }
+
+  .right ul {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 100%;
+  }
+
+  .right ul li {
+    width: 100%;
+  }
+
+  #about3 {
+    color: #091e42;
+    font-size: 40px;
+    margin-bottom: 10px;
+  }
+
+  .about4 {
+    color: #344563;
+    font-size: 17px;
+    margin-bottom: 10px;
+    padding-right: 70px;
+    margin-right: 30px;
+    word-spacing: 6px;
+    width: 87%;
+  }
+
+  .about5 {
+    color: #7f8cff;
+    word-spacing: 7px;
+    margin-bottom: 15px;
+  }
+
+  .about-links6 ul li {
+    list-style-type: none;
+    display: inline-block;
+    margin-right: 13px;
+    margin-left: 5px;
+  }
+
+  .about7 {
+    width: 155px;
+  }
+
+  .about9,
+  .about10,
+  .about11 {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: #344563;
+    font-size: 20px;
+    margin-bottom: 15px;
+    margin-top: 15px;
+  }
+
+  .big-rule {
+    width: 105%;
+    background-color: #dfe1e6;
+  }
+
+  .gray-box {
+    width: 122px;
+    height: 100px;
+    background-color: #f7f7f9;
+    cursor: pointer;
+    padding: 7px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .about12 {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 55%;
+    font-size: 15px;
+  }
+
+  .about12a {
+    width: 48px;
+    height: 48px;
+    margin-bottom: 10px;
+  }
+
+  .contact {
+    margin-top: 1px;
+    background: #6070ff;
+  }
+
+  .contact-box {
+    width: 30%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] We are very close to completing the desktop version of our application 🚶 . However, on this contact form, you can see that the `.contact-box` is too squeezed and therefore does not match the Figma design. I suggest that you modify the `width` so that your design matches the Figma.

You design 😎 
![image](https://user-images.githubusercontent.com/99394374/220840982-bd3b3033-c3ca-459f-b879-c685ca3f8a47.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/220841058-0ee4c4b0-850f-41aa-87ab-c983f82873cf.png)